### DISCUSSION TITLE
Old Reddit

### DISCUSSION THREAD

**Discussion by jechton** [catppuccin/catppuccin#2361] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Any additional comments?

There is currently only a userstyle for Reddit's new design, but many people still use [old.reddit.com](https://old.reddit.com).

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by isabelroses** [comment]
One of the issues with old.reddit.com, is that it can be themed by each subreddit, which makes it almost impossible to have a global theme through a userstyle.

Like compare https://old.reddit.com/r/gaming/ vs https://old.reddit.com/r/badUIbattles/ vs https://old.reddit.com/r/AITAH/
They are all insanely different which makes it incredibly difficult to manage, and would be hell to maintain, and probably worse then a google userstyle.

**Comment by kypwny** [comment]
its a good thing we solved that problem a long time ago
<img width="447" height="114" alt="image" src="https://github.com/user-attachments/assets/f5fe563b-673d-4af8-bec2-96047e47d486" />

**Comment by Boreo** [comment]
Opened a PR for this: catppuccin/userstyles#2331

The userstyles.yml note asks users to untick `allow subreddits to show me custom themes` in Reddit preferences.
